src/coreComponents/mesh/generators/VTKMeshGenerator.hpp

```diff
@@ -165,6 +166,9 @@ class VTKMeshGenerator : public ExternalMeshGeneratorBase
   /// Number of graph partitioning refinement iterations
   integer m_partitionRefinement = 0;
 
+  /// Whether global id arrays should be used, if available
+  integer m_useGlobalIds = 0;
```

Comment:
The default value of the field does not match the default value registered.
Is this intended?

Reply:
@TotoGaz No, this is the default value that should be used for backwards compatibility. I will change the default everywhere to `0`